SD-K9/src/app/pages/home/home.page.ts

```diff
@@ -3,10 +3,10 @@ import { MapCoordinator } from 'src/app/providers/map-coordinator.service';
 import { SVGManager } from 'src/app/providers/svg-manager.service';
 import { Location } from '../../helpers/location';
 import { ModalController, NavController } from '@ionic/angular';
-import { BusPage } from 'src/app/modals/bus/bus.page';
-import { AppsettingsPage } from 'src/app/modals/appsettings/appsettings.page';
+import { BusPage } from 'src/app/components/bus/bus.page';
+import { AppSettings } from 'src/app/pages/app-settings/app-settings.page';
 import { IonPullUpFooterState } from 'ionic-pullup';
-import { SVGCoordinate } from 'src/app/interfaces/svg-coordinate.model';
+import { SVGCoordinate } from 'src/app/models/svg-coordinate.model';
 
 declare var google;
```

Comment:
Is this necessary? I don't see it being used anywhere

Reply:
Is SVGCoordinate necessary? yes, floor-plan.component uses this to locate the selected classroom.
But if you refer back to the design updates, anything Coordinate and Location related will change. The changes depend on how the factory design and the routing pattern is done.
For now, to complete the file structure update, I had to refactor them as well.